style.css

```
@@ -0,0 +1,83 @@
+
+#logo {color: teal;
+font-family: monospace;
+font-size: 50px;
+font-style: italic;
+text-align: center;}
+
+nav {background-color: #DFEA9B;
+    }
```

It's best practice to have code formatted consistently throughout a project - use a linter such as Prettier to help, and work towards it as an example of what to aim for.

```suggestion
nav { 
    background-color: #DFEA9B;
}
```